### DISCUSSION TITLE
Illegal instruction exception

### DISCUSSION THREAD

**Discussion by rosekunthara** [gem5/gem5#1418] [Q&A] [answered]
Hi 
How can i change the privilege mode from user to machine in gem5 simulator to run a coremark benchmark as workload for riscv isa. I want to run the benchmark in SE mode specifically instead of FS mode.

**Comment by BobbyRBruce** [comment, marked as answer]
It cannot be done. SE mode cannot run privileged instructions. This is by design. If you want to run this you'll need to use FS mode.
